src/content/docs/en/guides/backend/xata.mdx

```diff
@@ -62,7 +62,7 @@ interface ImportMeta {
 Read more about [environment variables](/en/guides/environment-variables/) and `.env` files in Astro.
 :::
 
-Using the codegeneration from the Xata CLI and choosing the TypeScript option, generated an instance of the SDK for you, with types tailored to your database schema. Additionally, `@xata.io/client` was added to your `package.json`.
+Using the code generation from the Xata CLI and choosing the TypeScript option, generated an instance of the SDK for you, with types tailored to your database schema. Additionally, `@xata.io/client` was added to your `package.json`.
```

Comment:
Just checking that you're sure this isn't some proper name/term!

Reply:
Just checked on this, and looks like it's the `codegen` CLI command, so yes, this should just be in regular English words!